pkg/graph/resolvers/governance_request_feedback.go

```diff
@@ -20,7 +21,7 @@ func GetGovernanceRequestFeedbackAuthor(
 	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
 	feedbackAuthorEUAID string,
 ) (*models.UserInfo, error) {
-	authorInfo, err := fetchUserInfo(ctx, feedbackAuthorEUAID)
+	authorInfo, err := dataloaders.GetUserInfo(ctx, feedbackAuthorEUAID)
```

Comment:
I think this is good to use the dataLoader. 

Tangential and out of scope of this ticket, but I think we could / and should set up the service initialization in routes to use the dataloader. 

So anytime fetchUserInfo is called, we are using the dataloader version.

Reply:
Can you create a tech debt ticket for that? Or do it as a NOREF PR, and check that it works?

Reply:
[EASI-3167](https://jiraent.cms.gov/browse/EASI-3167)